data/doveadm.js

```diff
@@ -2075,195 +2376,257 @@ returned.`,
 				type: doveadm_arg_types.STRING
 			},
 			name: {
+				example: 'metric_name',
 				positional: true,
-				type: doveadm_arg_types.STRING
+				type: doveadm_arg_types.STRING,
+				text: `Metric name.`,
 			},
 			filter: {
 				positional: true,
 				type: doveadm_arg_types.STRING
 			},
 		},
-		man: 'doveadm-stats'
+		man: 'doveadm-stats',
+		text: `Add metrics to statistics.`,
 	},
 
 	'stats dump': {
 		args: {
 			socketPath: {
 				cli: 's',
 				cli_only: true,
-				type: doveadm_arg_types.STRING
+				example: '/path/to/stats',
+				type: doveadm_arg_types.STRING,
+				text: `Path to socket path.`,
 			},
 			reset: {
 				cli: 'r',
-				type: doveadm_arg_types.BOOL
+				type: doveadm_arg_types.BOOL,
+				text: `Reset stats after dumping.`,
 			},
 			fields: {
 				cli: 'f',
-				type: doveadm_arg_types.STRING
+				type: doveadm_arg_types.STRING,
+				text: `Stats fields to return.`,
 			},
 		},
-		man: 'doveadm-stats'
+		man: 'doveadm-stats',
+		text: `Output statistics.`,
 	},
 
 	'stats remove': {
 		args: {
 			name: {
+				example: 'metric_name',
 				positional: true,
-				type: doveadm_arg_types.STRING
+				type: doveadm_arg_types.STRING,
+				text: `The metric to remove.`,
 			},
 		},
-		man: 'doveadm-stats'
+		man: 'doveadm-stats',
+		text: `Remove metrics from statistics.`,
 	},
 
 	'stats reopen': {
 		args: {},
-		man: 'doveadm-stats'
+		man: 'doveadm-stats',
+		text: `Reopen file exporter files.`,
 	},
 
 	'stop': {
 		args: {},
-		// TODO: man page needed
+		man: 'doveadm',
+		text: `Stop Dovecot.`,
 	},
 
 	sync: {
 		args: {
 			'full-sync': {
 				cli: 'f',
 				type: doveadm_arg_types.BOOL,
+				text: `Do full synchronization.`,
 			},
 			'purge-remote': {
 				cli: 'P',
 				type: doveadm_arg_types.BOOL,
+				text: `Run a purge for remote after sync.`,
 			},
 			'reverse-sync': {
 				cli: 'R',
 				type: doveadm_arg_types.BOOL,
+				text: `Do a reverse sync.`,
 			},
 			'lock-timeout': {
 				cli: 'l',
+				example: 60,
 				type: doveadm_arg_types.INTEGER,
+				text: `Lock timeout (in seconds).`,
 			},
 			rawlog: {
 				cli: 'r',
+				example: '/tmp/rawlog',
 				type: doveadm_arg_types.STRING,
+				text: `Rawlog path.`,
 			},
 			mailbox: {
 				cli: 'm',
+				example: 'INBOX',
 				type: doveadm_arg_types.STRING,
+				text: `Only sync this mailbox.`,
```

Comment:
no question marks here?

Reply:
These are intended to document state.  The command has a default way of doing the action, if you set the boolean to true it changes that state.  We aren't really asking the user a question.